karma.conf.js

```diff
@@ -81,7 +81,7 @@ module.exports = function init(config) {
     customLaunchers: {
       ChromeHeadlessNoSandbox: {
         base: 'ChromeHeadless',
-        flags: ['--no-sandbox'],
+        flags: ['--no-sandbox', '--ignore-gpu-blacklist'],
```

Comment:
@jourdain this seems to do the trick for WebGL CI tests :fireworks:

Reply:
This is great... I was waiting for chrome to fully work headless with webgl.